[2025-01-07T08:06:05.534+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_file_processing.etl_process_ih manual__2025-01-07T08:06:00.318282+00:00 [queued]>
[2025-01-07T08:06:05.539+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_file_processing.etl_process_ih manual__2025-01-07T08:06:00.318282+00:00 [queued]>
[2025-01-07T08:06:05.540+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2025-01-07T08:06:05.548+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): etl_process_ih> on 2025-01-07 08:06:00.318282+00:00
[2025-01-07T08:06:05.555+0000] {standard_task_runner.py:57} INFO - Started process 522 to run task
[2025-01-07T08:06:05.558+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'etl_file_processing', 'etl_process_ih', 'manual__2025-01-07T08:06:00.318282+00:00', '--job-id', '292', '--raw', '--subdir', 'DAGS_FOLDER/ETL_file_processing.py', '--cfg-path', '/tmp/tmps8zcdwc3']
[2025-01-07T08:06:05.559+0000] {standard_task_runner.py:85} INFO - Job 292: Subtask etl_process_ih
[2025-01-07T08:06:05.621+0000] {task_command.py:410} INFO - Running <TaskInstance: etl_file_processing.etl_process_ih manual__2025-01-07T08:06:00.318282+00:00 [running]> on host 8fff7f1050bf
[2025-01-07T08:06:05.685+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_file_processing' AIRFLOW_CTX_TASK_ID='etl_process_ih' AIRFLOW_CTX_EXECUTION_DATE='2025-01-07T08:06:00.318282+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-07T08:06:00.318282+00:00'
[2025-01-07T08:06:06.892+0000] {logging_mixin.py:150} INFO - No data found in the 'ihmodels' collection.
[2025-01-07T08:06:06.893+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-01-07T08:06:06.905+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=etl_file_processing, task_id=etl_process_ih, execution_date=20250107T080600, start_date=20250107T080605, end_date=20250107T080606
[2025-01-07T08:06:06.957+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-01-07T08:06:06.992+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
